requirements/dev.txt

```diff
@@ -3,5 +3,5 @@
 
 django-debug-toolbar
 sphinx
-django>=1.4,<1.10
```

Comment:
@brad Oh yeah, good point, these won't be installed by projects that use this app, so there isn't much risk. It still seems like the upper limits are useful though. I'm thinking of the scenario where a new contributor sets up their virtualenv to run the tests. An upgrade to a future version of `mock` or `freezegun` could cause tests to fail.